epilogue-processor/src/main/java/edu/wpi/first/epilogue/processor/ElementHandler.java

```diff
@@ -61,14 +62,24 @@ protected TypeMirror dataType(Element element) {
    * @return the name specified in the {@link Logged @Logged} annotation on the element, if present;
    *     otherwise, the field or method's name with no modifications
    */
-  public String loggedName(Element element) {
+  public static String loggedName(Element element) {
     var elementName = element.getSimpleName().toString();
     var config = element.getAnnotation(Logged.class);
 
     if (config != null && !config.name().isBlank()) {
       return config.name();
     } else {
-      return elementName;
+      // Delete common field prefixes (k_Name, m_name, s_name)
+      elementName = elementName.replaceFirst("^[msk]_", "");
+      if (elementName.matches("^k[A-Z]")) {
```

Comment:
If Java's default regex engine supports lookaheads, this could be a replace with lookahead rather than an if statement.

Reply:
This was to avoid replacing the first capital letter when this was part of the group above, eg `"kName".replaceFirst("^k[A-Z]", "")` would yield `"ame"` instead of `"Name"`. Capture groups could also work, eg `"kName".replaceFirst("^k([A-Z])", "$1")`

Reply:
That's why the lookahead -- `^k(?=[A-Z])` should work properly. Lookaheads match if the content is there but doesn't consume/include it into the match (and therefore won't be replaced).
Only risk is that some regex engines don't support lookaheads; I think Java's does (I saw some other Epilogue code using lookaheads).